src/mockttp.ts

```diff
@@ -576,7 +593,8 @@ export type SubscribableEvent =
     | 'response'
     | 'abort'
     | 'tls-client-error'
-    | 'client-error';
+    | 'client-error'
+    | 'handle-error';
```

Comment:
I think this should be `handler-error` - that's slightly clearer (since `handle` has other meanings, like file handles). This is basically an event you can subscribe to to hear about any request handlers which throw errors at any point during request processing.